### ISSUE TITLE
Add jobTitle to https://schema.org/Person

### ISSUE THREAD

**Issue by chrispauly** [RehanSaeed/Schema.NET#686]
### Describe the feature

The property is defined on schema.org, but I don't see it on the Schema.NET Person object.  https://webschemas.appspot.com/jobTitle

### Schema objects

https://schema.org/Person

**Comment by SmithPlatts** [comment]
As per the docs:

> This term is in the "new" area - implementation feedback and adoption from applications and websites can help improve our definitions.

It should be present in the `Schema.NET.Pending` package.

**Comment by chrispauly** [comment]
@SmithPlatts Thank you. I had a difficult time figuring out what is considered pending.  I tried switching my project to Pending and didn't see it.  But I see the mention now in Schema.org referring to that property being considered new.  And I found a similar version of this question/issue asked a while ago. https://github.com/RehanSaeed/Schema.NET/issues/423

**Comment by Turnerj** [comment]
Hey @chrispauly - if it is missing from Schema.NET.Pending, I'll do a new release with a newer version of the JSON-LD in a few days.

Thanks for pointing Chris in the right direction @SmithPlatts.

**Comment by chrispauly** [comment]
@Turnerj my testing was quick and maybe I didn't test it correctly.  I just switched my packages.config package name and I can't even remember if I restored nugets. I thought I had, but may not have. I checked the code after making that swap and didn't see the property... but may've gotten confused. It's a daytime job and since it's the weekend for me... I won't be able to check again until Monday. I don't want you to do extra work if I just made a mistake.
